test/core/stake/totalWeightAtHeight.js

```diff
@@ -91,10 +91,10 @@ contract('Stake.totalWeightAtHeight()', async (accounts) => {
         let expectedWeights = [
             {height: 0, totalWeight: new BN('2')},
             {height: 1, totalWeight: new BN('2')},
-            {height: 2, totalWeight: new BN('5')},
-            {height: 3, totalWeight: new BN('9')},
-            {height: 4, totalWeight: new BN('33')},
-            {height: 10, totalWeight: new BN('33')},
+            {height: 2, totalWeight: new BN('2')},
+            {height: 3, totalWeight: new BN('5')},
+            {height: 4, totalWeight: new BN('9')},
+            {height: 6, totalWeight: new BN('33')},
```

Comment:
Please also check for height `5`
```suggestion
            {height: 5, totalWeight: new BN('33')},
            {height: 6, totalWeight: new BN('33')},
```